front/src/applications/rollingStockEditor/views/RollingStockEditor.tsx

```diff
@@ -89,6 +100,9 @@ const RollingStockEditor = () => {
                     id={openedRollingStockCardId}
                     isEditing={isEditing}
                     rollingStock={selectedRollingStock}
+                    exportRollingStock={exportRollingStock}
+                    exportTranslation={t('rollingStock.export')}
+                    exportRollingStockTranslation={t('rollingStock.exportRollingStock')}
```

Comment:
You can remove these 3 props and instantiate them directly in `RollingStockInformationPanel`